### DISCUSSION TITLE
Searching phrases, metadata in windows search

### DISCUSSION THREAD

**Discussion by jesuslop** [Flow-Launcher/Flow.Launcher#2678] [General]
Hi, not sure if this is proper to post. I'm trying to do a file content search looking for a phrase ("potential theory") in a bunch of pdfs. In windows explorer search bar it works nicely. I double qoute the phrase "potential theory", kick enter, and there they are. With Flow Launcher, it works OK searching for single words, but for a phrase like that, the search returns too many results as if searching for pdfs including "potential" somewhere and "theory" elsewhere. Search by metadata works also on explorer and not in FL. (for instance a search authors:xxx or keywords:kkk) or boolean operators. It would be nice (or even a must have) to have some way of working with windows advanced query syntax, https://learn.microsoft.com/en-us/windows/win32/lwef/-search-2x-wds-aqsreference. Thanks!

**Comment by jjw24** [comment]
Hi there, I converted this into issue #2804 :) closing this one.
